client-admin/src/components/conversation-admin/conversation-config.js

```diff
@@ -166,7 +166,8 @@ class ConversationConfig extends React.Component {
           <Box sx={{ flexShrink: 0, position: "relative", top: -0.5 }}>
             <input
               type="checkbox"
-              label="Visualization"
+              aria-label="Visualization"
+              id="vis_type"
```

Comment:
Added IDs to allow e2e tests

Reply:
Do you prefer ids to data-test- attributes?

Reply:
Replaced these with `data-test-id` 🎉 Thanks!